tools/jbuild

```diff
@@ -0,0 +1,13 @@
+(jbuild_version 1)
+
+(library
+ ((name tyxml_tools)
+  (public_name tyxml.tools)
+  (wrapped false)
+  (modules tyxml_name)
+  (libraries (bytes))))
```

Comment:
There's no need to ever use the `bytes` lib with jbuilder. This library is only useful for building on `<4.02` which you can't do anyway with jbuilder. On the other hand, it forces you to declare a dependency on `base-bytes` which depend on `findlib`. A big negative IMO.

Reply:
Fair enough!